Blueprints/BlueprintDefinitions/vs2022/MessageProcessingFramework/template/src/BlueprintBaseName.1/Functions.cs

```diff
@@ -0,0 +1,92 @@
+using Amazon.Lambda.Annotations;
+using Amazon.Lambda.Annotations.APIGateway;
+using Amazon.Lambda.Core;
+using Amazon.Lambda.SQSEvents;
+using AWS.Messaging;
+using AWS.Messaging.Lambda;
+
+// Assembly attribute to enable the Lambda function's JSON input to be converted into a .NET class.
+[assembly: LambdaSerializer(typeof(Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer))]
+
+namespace BlueprintBaseName._1;
+
+public class Functions
+{
+    /// <summary>
+    /// This is the component of the AWS Message Processing Framework for .NET that dispatches
+    /// messages in the <see cref="SQSEvent"/> received from the Lambda service to the correct 
+    /// handler for each message based on its type
+    /// </summary>
+    private ILambdaMessaging _lambdaMessaging;
+
+    /// <summary>
+    /// Default constructor. This constructor is used by Lambda to construct the instance. When invoked in a Lambda environment
+    /// the AWS credentials will come from the IAM role associated with the function and the AWS region will be set to the
+    /// region the Lambda function is executed in.
+    /// </summary>
+    /// <param name="lambdaMessaging">Framework component that processes messages in Lambda</param>
+    public Functions(ILambdaMessaging lambdaMessaging)
```

Comment:
You could be modern and use [primary constructors](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/instance-constructors#primary-constructors).

No big deal if you don't want to and save that effort to another time of going through the other annotations based templates.